src/main/webapp/dev/js/common/feedbackResponseComments.es6

```diff
@@ -201,6 +345,12 @@ const addCommentHandler = (e) => {
                 addFormRow.prev().show();
                 addFormRow.hide();
                 destroyEditor(`responseCommentAddForm-${responseCommentId}`);
+                if (isOnQuestionsPage != null && isOnQuestionsPage) {
+                    const recipientIndex = responseCommentId.substring(0, 1);
+                    const giverIndex = responseCommentId.substring(2, 3);
+                    const questionIndex = responseCommentId.substring(4, 5);
```

Comment:
The code here seems to assume that the index is always comprised of a single digit, which may not be the case in actual usage. This results in the add comment form not appearing after successfully adding a comment.

![panel2](https://user-images.githubusercontent.com/7546744/29747251-389be560-8b26-11e7-89bc-b6eb466c7977.png)